deleteacls_test.go

```diff
@@ -0,0 +1,91 @@
+package kafka
+
+import (
+	"context"
+	"testing"
+
+	ktesting "github.com/segmentio/kafka-go/testing"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClientDeleteACLs(t *testing.T) {
```

Comment:
Is there any value in querying for a deleted ACL and expecting an empty/"not found" response in addition to checking for non-error in deleteResp?

Reply:
Good idea, I just added this